packages/@react-types/actionbar/src/index.d.ts

```diff
@@ -10,14 +10,15 @@
  * governing permissions and limitations under the License.
  */
 
-import {CollectionBase, DOMProps, StyleProps} from '@react-types/shared';
-import {Key, ReactNode} from 'react';
+import {CollectionBase, DOMProps, DOMRefValue, StyleProps} from '@react-types/shared';
+import {Key, ReactNode, RefObject} from 'react';
 
 export interface ActionBarProps<T> extends CollectionBase<T> {
   // Required. When zero, the ActionBar is hidden.
   selectedItemCount: number | 'all',
   // Also required. When clicked, the selection should be cleared.
   onClearSelection: () => void,
+  restoreFocusRef: RefObject<DOMRefValue<HTMLElement>> | RefObject<HTMLElement>,
```

Comment:
API change I'm not sure about

Reply:
IMO we should remove this and relocate it into the `ActionBarInnerProps` since it is only used for ActionBarInner and shouldn't be visible to external users

Reply:
o, good catch, didn't notice that due to the code folding that github does, that is what we should do